### ISSUE TITLE
Draggable attribute always true

### ISSUE THREAD

**Issue by marvinhagemeister** [preactjs/preact#2208]
Reported via slack. Haven't verified if this is an actual issue yet.

> Anders Riggelsen 12:47 AM
I have a weird bug with Preact. A `draggable` attribute on an element always seem to be flipped to`true` in the DOM despite I can step through the JS code and inspect that in fact I generate a value of `false` to be used. Anyone have any idea where to start?
>
> Hmm seems to be a bug in Preact. Instead of setting the value to `false` I had to set it to null for it to be set to `false` in the DOM.
Very weird behavior.

**Comment by marvinhagemeister** [comment]
This one may be related and can likely be closed if this here is fixed: https://github.com/preactjs/preact/issues/1849

**Comment by jeremy-coleman** [comment]
Did u compare =false vs =“false” ?

**Comment by marvinhagemeister** [comment]
Just checked this and I think I know where the confusion comes from. We set the correct value for `draggable` by calling `node.setAttribute("draggable", value)`. Both Chrome, Firefox and probably other browsers too, will display the stringified boolean value in the DOM using that method.

Curiously there is a section in the spec that specifically says the browser behavior should be treated as being invalid. See this section on [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#Boolean_Attributes):

> To be clear, the values "true" and "false" are not allowed on boolean attributes. To represent a false value, the attribute has to be omitted altogether. This restriction clears up some common misunderstandings: With checked="false" for example, the element’s checked attribute would be interpreted as true because the attribute is present.

The current implementation in browsers contradicts that statement :woman_shrugging: In either way, this is only a matter of what's displayed in the elements panel. Preact sets the `draggable` value correctly and it works in all browser I checked.
